webpack_loader/loader.py

```diff
@@ -39,15 +39,23 @@ def get_assets(self):
         return self.load_assets()
 
     def filter_chunks(self, chunks):
-        assets = self.get_assets()
+        filtered_chunks = []
 
         for chunk in chunks:
             ignore = any(regex.match(chunk)
                          for regex in self.config['ignores'])
             if not ignore:
-                files = assets['assets']
-                url = self.get_chunk_url(files[chunk])
-                yield { 'name': chunk, 'url': url }
+                filtered_chunks.append(chunk)
+
+        return filtered_chunks
+
+    def map_chunk_files_to_url(self, chunks):
+        assets = self.get_assets()
+
+        for chunk in chunks:
+            files = assets['assets']
```

Comment:
I think it makes more sense for this line to be out of the `for`

Reply:
Done